### ISSUE TITLE
compatible to tensorflow v1.4.0

### ISSUE THREAD

**Issue by neuhzhj2012** [AliAbbasi/Multilabel-Image-Classification-with-Softmax#1]
modify  two tensorflow functions:
------label_split = tf.split( 1, 2, self.y_     ) 
+++label_split = tf.split( self.y_, 2, 1  )
------score_split = tf.split( 1, 2, self.score_ )
+++score_split = tf.split( self.score_, 2, 1 )

----total += tf.reduce_mean(tf.nn.softmax_cross_entropy_with_logits( score_split[i] ,label_split[i] ))
+++total += tf.reduce_mean(tf.nn.softmax_cross_entropy_with_logits(logits= score_split[i] ,labels= label_split[i] ))

**Comment by AliAbbasi** [comment]
Thanks for your comment, as I mentioned in README, this code is for Tensorflow 1.1.
